Tests/MongoSwiftSyncTests/SpecTestRunner/SpecTest.swift

```diff
@@ -220,6 +212,16 @@ extension SpecTestFile {
                 continue
             }
 
+            // abort any open transactions from previous test runs
```

Comment:
if this is working maybe it doesn't matter, but the unified test runner has you run this with primary read preference in repl sets, and run it against every mongos individually. if only the existing spec tests had all been ported to the new format, so you wouldn't have to update this runner...

https://github.com/mongodb/mongo-swift-driver/blob/8e7a0ff92f5dee4de5c5fb365eaacd7ab8206ebf/Tests/MongoSwiftSyncTests/UnifiedTestRunner/UnifiedTestRunner.swift#L24

Reply:
Since this runner will be deleted eventually anyways, I just copied the implementation over from the unified runner.